debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9IndexableObjectHelper.java

```diff
@@ -120,7 +135,26 @@ public static U32 size(J9ObjectPointer objPointer) throws CorruptDataException
 		return size(J9IndexableObjectPointer.cast(objPointer));
 	}
 
+	/**
+	 * Get the header size of the contiguous indexable object.
+	 */
 	public static long contiguousHeaderSize()
+	{
+		try {
+			if (!J9BuildFlags.env_data64 || J9RASHelper.getVM(DataType.getJ9RASPointer()).isIndexableDataAddrPresent().isZero()) {
+				return contiguousHeaderSizeLegacy();
+			} else {
+				return contiguousHeaderSizeWithDataAddress();
+			}
+		} catch (NoSuchMethodError | NoSuchFieldException | CorruptDataException e) {
+			return contiguousHeaderSizeLegacy();
+		}
```

Comment:
Suggest inverting this so we only need one place that calls `contiguousHeaderSizeLegacy()`.

Reply:
Perhaps I am missing something, but won't inverting this still result in two calls to `contiguousHeaderSizeLegacy()`, or would you be suggesting that the `catch` block be removed?